app/routes/webhooks.tsx

```diff
@@ -10,7 +10,7 @@ export const action = async ({ request }: ActionFunctionArgs) => {
     throw new Response();
   }
 
-  switch (topic) {
+  switch (topic as string) {
```

Comment:
Temporary solution until https://github.com/Shopify/shopify-app-js/pull/1047 gets released, so we can update the dependency and topic will already be a string.